.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/

```

Comment:
- [ ] As, previously mentioned by the reviewer, please add test.md here.

Reply:
Ok